be1-go/channel/authentication/authentication_test.go

```diff
@@ -158,6 +158,9 @@ func Test_Authenticate_User(t *testing.T) {
 	err = authCha.Publish(msg, socket.ClientSocket{})
 	require.NoError(t, err)
 
+	// The websocket server may take some time to receive and process a message
+	time.Sleep(time.Millisecond * 100)
```

Comment:
Thanks for taking the initiative to fix this test ! Unfortunately, adding arbitrary delays in tests is not only unreliable (e.g. false negatives) but can also slow down the pipeline if it is done often (which is the case for our system).
You could fix this by using channels and checking the boolean once you know the value has been received (you can also add a timeout if you're not sure the check can be reached).  
I'm aware that using `time.Sleep` is common in our codebase but it's probably something we should refactor at some point. 
Let me know if you need clarifications from my part :)